### DISCUSSION TITLE
Android TV/Google TV

### DISCUSSION THREAD

**Discussion by skiely** [LivingWithHippos/unchained-android#111] [Q&A]
Any plans to have it work on Android/Google TV?

**Comment by LivingWithHippos** [comment]
I checked how to do it this morning, the problem was that writing for android tv could require a lot of coding (not worth it for me). Apparently I could create a new module to share all the code I want between the android tv and the phone version. I'll have to try and check how much work it is to adapt it.

**Reply by skiely** [reply]
Ok cool thanks anyway. Loving the app so far..
